examples/nylas-connect/app.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+import json
+import sys
+import os
+import flask
+import requests
+import urllib
+import logging
+import subprocess
+
+from nylas import APIClient
+
+logging.basicConfig(format='%(asctime)s %(levelname)s: %(message)s')
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+try:
+    from credentials import *
+except ImportError:
+    log.error("Couldn't import credentials.py --- you'll need to create it.")
+    log.error("See credentials.py.template for more details.")
+    sys.exit(-1)
+
+app = flask.Flask(__name__)
+
+# These are the permissions your app will ask the user to approve for access
+SCOPE = ' '.join(['https://mail.google.com/',
+                  'https://www.googleapis.com/auth/calendar',
+                  'https://www.googleapis.com/auth/userinfo.email',
+                  'https://www.googleapis.com/auth/userinfo.profile',
+                  'https://www.googleapis.com/auth/calendar',
+                  'https://www.google.com/m8/feeds/'])
+
+# This is the path in your Google application that users are redirected to after they
+# have authenticated with Google, and it must be authorized through Google's
+# developer console
+REDIRECT_URI = '' # Note: Use your ngrok url here if testing locally
```

Comment:
Explanation about ngrok? Also you could tell people they can edit their /etc/hosts file to have the redirect work.